packages/schematics/angular/ng-new/schema.json

```diff
@@ -125,10 +125,11 @@
         "message": "Which stylesheet format would you like to use?",
         "type": "list",
         "items": [
-          { "value": "css",  "label": "CSS    (.css )" },
-          { "value": "sass", "label": "Sass   (.scss) [ http://sass-lang.com   ]" },
-          { "value": "less", "label": "Less   (.less) [ http://lesscss.org     ]" },
-          { "value": "styl", "label": "Stylus (.styl) [ http://stylus-lang.com ]" }
+          { "value": "css",  "label": "CSS" },
+          { "value": "scss", "label": "SCSS   [ http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax ]" },
```

Comment:
opt suggestion: add parenthesis saying `(css-style syntax)` `(indented syntax)`

Reply:
I am still not sure about the added value of the above tbh. And I fear more confusion with introduction of new terms.